### DISCUSSION TITLE
Migration tool for the old version of IOTA

### DISCUSSION THREAD

**Discussion by LO-H-W** [iotaledger/legacy-migration-tool#148] [Help & Support] [answered]
Backup text: Enter your IOTA seed (81 characters).
I can't find where my seed is displayed.

File backup: Import the Trinity SeedVault file (with the extension .kdbx).
I tried to back up, but the file doesn't end with .kdbx.

Here are the device error prompts:

App Version: 1.7.1

platform: win32
platformVersion: 10.0.19045
platformArchitecture: x64
cpuCount: 2
totalMem: 3918.8 MB
freeMem: 955.9 MB

Error Type: [Main Context] Unhandled Rejection

Error: Cannot find latest.yml in the latest release artifacts (https://github.com/iotaledger/legacy-migration-tool/releases/download/v1.0.0/latest.yml): HttpError: 404 
"method: GET url: https://github.com/iotaledger/legacy-migration-tool/releases/download/v1.0.0/latest.yml\n\nPlease double check that your authentication token is correct. Due to security reasons actual status maybe not reported, but 404.\n"
Headers: {
  "server": "GitHub.com",
  "date": "Sat, 21 Dec 2024 03:33:54 GMT",
  "content-type": "text/plain; charset=utf-8",
  "vary": "X-PJAX, X-PJAX-Container, Turbo-Visit, Turbo-Frame, Accept-Encoding, Accept, X-Requested-With",
  "cache-control": "no-cache",
  "strict-transport-security": "max-age=31536000; includeSubdomains; preload",
  "x-frame-options": "deny",
  "x-content-type-options": "nosniff",
  "x-xss-protection": "0",
  "referrer-policy": "no-referrer-when-downgrade",
  "content-security-policy": "default-src 'none'; base-uri 'self'; connect-src 'self'; form-action 'self'; img-src 'self' data:; script-src 'self'; style-src 'unsafe-inline'",
  "content-encoding": "gzip",
  "content-length": "29",
  "x-github-request-id": "6C6E:2E3F78:1F820E:2AF02B:67663722"
}
    at d (C:\Users\H\AppData\Local\Programs\desktop\resources\app.asar\public\build\main.js:2:184493)
    at a.handleResponse (C:\Users\H\AppData\Local\Programs\desktop\resources\app.asar\public\build\main.js:2:186132)
    at ClientRequest.<anonymous> (C:\Users\H\AppData\Local\Programs\desktop\resources\app.asar\public\build\main.js:2:185593)
    at ClientRequest.emit (events.js:315:20)
    at ClientRequest.EventEmitter.emit (domain.js:467:12)
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:105:6816)
    at SimpleURLLoaderWrapper.emit (events.js:315:20)
    at SimpleURLLoaderWrapper.EventEmitter.emit (domain.js:467:12)
    
    Please help me transfer the IOTA Mi from my old device to the latest version.

**Comment by begonaalvarezd** [comment, marked as answer]
This legacy tool is to migrate from legacy iotas to the new network, in order to do so you need an old seedvault file (.kdbx) or a legacy seed which is 81 characters. If you dont have any of these, you cant migrate (unless you used ledger, if thats the case you should migrate using your ledger)

It might be that you dont need migration, if you have a 24  words mnemonic or a stronghold file then you dont need migration. 

Please visit out official discord https://discord.iota.org/ if you need further support, github is not a support platform
